File: arrays-and-strings/01-is-unique/java/SolutionHashSet.java
import java.util.HashSet;

public class SolutionHashSet {

    static boolean isUnique(String text) {
        HashSet<Character> characters = new HashSet<>();

        for (char c : text.toCharArray()) {
            if (!characters.add(c)) {
                return false;
            }
        }

        return true;
    }

    public static void main(String[] args) {
        System.out.println(isUnique("LoremIpsum"));
    }
}

File: arrays-and-strings/01-is-unique/csharp/SolutionHashSet.cs
using System;
using System.Collections.Generic;

public class Program
{
    static bool IsUnique(string text)
    {
        HashSet<char> characters = new();

        foreach (char c in text)
        {
            if (!characters.Add(c))
            {
                return false;
            }
        }

        return true;
    }

    static void Main(string[] args)
    {
        Console.WriteLine(IsUnique("LoremIpsu"));
    }
}